application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -75,6 +75,22 @@
   #set ($keyboardShortcutsEnabled = false)
   #set ($discard = $xwiki.ssx.use('Diagram.DiagramSheet'))
   #set ($discard = $xwiki.jsx.use('Diagram.DiagramEditSheet'))
+  ## Issue #219: Cannot modify diagram title in diagram editor
+  &lt;div class="row xform"&gt;
+    &lt;div class="col-xs-12 col-md-12"&gt;
+      &lt;dl&gt;
+        &lt;dt&gt;
+          &lt;label for="xwikiDocTitleInput"&gt;
+            $services.localization.render('core.editors.content.titleField.label')
```

Comment:
```suggestion
            $escapetool.xml($services.localization.render('core.editors.content.titleField.label'))
```

Always escape content that you don't control directly. In this case, the translation message can easily break the HTML.